changelog/2021-10-26T13_47_06+02_00_add_tdp_bram_enables

```diff
@@ -0,0 +1,2 @@
+INTERNAL CHANGE: Added NoOp to RamOp for `Clash.Explicit.BlockRam.trueDualPorBlockRam`,
+NoOp pulls the enable signal of trueDualPortBlockRam low for the relevant port.
```

Comment:
This is an API change, right? Why is it marked "internal"?

Reply:
Since tdprams haven't been part of a release, just update the changelog that introduced them